core/profiles/standard/standard.install

```diff
@@ -171,14 +192,78 @@ function standard_install() {
     ->set('node_post_pattern', 'posts/[node:title]')
     ->save();
 
-  // Create an initial post to welcome users on install.
+  // Create three default Promobox nodes.
```

Comment:
If we are to call these "Cards", then this inline comment should also be updated to reflect that:

```suggestion
  // Create three default Card nodes.
```